roffit

```diff
@@ -256,6 +256,33 @@ sub linkfile {
             # convert https://, http:// and ftp:// URLs to <a href> links
             $field =~ s/(^|\W)((https|http|ftp):\/\/[a-z0-9\-._~%:\/?\#\[\]\@!\$&'()*+,;=]+)/$1<a href=\"$2\">$2<\/a>/gi;
 
+            # remove punctuation characters at end of url
+            $field =~ s/(href=\")([^\"]*)(&quot;)/$1$2/g;
```

Comment:
This does not remove punctuation. What is this for?

Reply:
\&quot; seemed to be a troublemaker so for following lines to work it had to be removed first in process.